admissioncontroller/whorf.py

```diff
@@ -31,10 +40,16 @@ def validating_webhook():
         webhook.logger.error('K8s UID failed security checks. Request rejected!')
         return admission_response(False, uid, response)
 
+    # check we're not in the kube-system namespace
+    namespace = request_info["request"].get("namespace")
+    if namespace in ignore_list:
+        print("Namespace in ignore list. Ignoring validation")
```

Comment:
use the logger over print

Reply:
Both are present.  Shall I delete print anyway?